src/ontology/mondo-edit.obo

```diff
@@ -418814,16 +418814,11 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 [Term]
 id: MONDO:0025510
 name: pythiosis, non-human animal
-def: "A granulomatous disease caused by the aquatic organism pythium insidiosum and occurring primarily in horses, cattle, dogs, cats, fishes, and rarely in humans. It is classified into three forms: ocular, cutaneous, and arterial." [MESH:D058968]
-synonym: "human pythiosis" NARROW [GARD:0011989]
-synonym: "pythium insidiosum infection" RELATED [GARD:0011989]
-xref: EFO:1001410 {source="MONDO:equivalentTo"}
-xref: GARD:0011989 {source="MONDO:equivalentTo"}
-xref: MESH:D058968 {source="MONDO:equivalentTo"}
-xref: UMLS:C0276912 {source="MONDO:equivalentTo"}
+def: "A granulomatous disease caused by the aquatic organism pythium insidiosum and occurring primarily in horses, cattle, dogs, cats, and fishes. It is classified into three forms: ocular, cutaneous, and arterial." [MESH:D058968]
```

Comment:
@katiermullen, please add your orcid here in addition to MESH. This way we know you have tweaked the definition (ie it is not the exact same as MESH)

Reply:
Done!